interrupts/README.md

```diff
@@ -1,6 +1,6 @@
 # Interrupts and Interrupt Handling
 
-You will find a couple of posts which describes an interrupts and an exceptions handling in the linux kernel.
+You will find a couple of posts which describe an interrupts and an exceptions handling in the linux kernel.
```

Comment:
This sentence is malformed, you can fix it entirely with "You will find a couple of posts which describe interrupts and exceptions handling in the linux kernel."